monocle/db/queries.py

```diff
@@ -759,3 +764,44 @@ def new_contributors(es, index, repository_fullname, params):
     )
     diff = new.difference(old)
     return {'items': [n for n in new_authors if n['key'] in diff]}
+
+
+def changes_by_file_map(es, index, repository_fullname, params):
```

Comment:
Could you add the support for size. And also order the returned values.

Reply:
the values are in a dict so there is no need for sorting.